src/main/java/uk/gov/hmcts/cmc/claimstore/models/ccj/RepaymentPlan.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.cmc.claimstore.models.ccj;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+import static uk.gov.hmcts.cmc.claimstore.utils.ToStringStyle.ourStyle;
+
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class RepaymentPlan {
+
+    public enum PaymentSchedule {
+        EACH_WEEK,
```

Comment:
`EVERY_WEEK` for consistency?

Reply:
I reused what we have in cmc citizen frontend